specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -6,127 +6,129 @@ This specification refers to the following
 
-_Jakarta™ Server Pages Specification, Version 2.3_. Available at: _https://jakarta.ee/specifications/pages/2.3_
+_Jakarta™ Enterprise Beans Specification, Version 4.0_. Available at: _https://jakarta.ee/specifications/enterprise-beans/4.0_
 
-_Jakarta™ Expression Language Specification, Version 3.0_. Available at: _https://jakarta.ee/specifications/expression-language/3.0_
+_Jakarta™ Server Pages Specification, Version 3.0_. Available at: _https://jakarta.ee/specifications/pages/3.0_
 
-_Jakarta™ Annotations Specification, Version 1.3_. Available at: _https://jakarta.ee/specifications/annotations/1.3_
+_Jakarta™ Expression Language Specification, Version 4.0_. Available at: _https://jakarta.ee/specifications/expression-language/4.0_
 
-_Jakarta™ Servlet Specification, Version 4.0_. Available at: _https://jakarta.ee/specifications/servlet/4.0_
+_Jakarta™ Annotations Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/annotations/2.0_
+
+_Jakarta™ Servlet Specification, Version 5.0_. Available at: _https://jakarta.ee/specifications/servlet/5.0_
 
 _JDBC™ 4.2 API (JDBC specification)_. Available at: _https://jcp.org/en/jsr/detail?id=221_
 
 _Java™ Naming and Directory Interface 1.2 Specification (JNDI specification)_. Available at: _https://docs.oracle.com/javase/8/docs/technotes/guides/jndi/index.html_
 
-_Jakarta™ Messaging Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/messaging/2.0_
+_Jakarta™ Messaging Specification, Version 3.0_. Available at: _https://jakarta.ee/specifications/messaging/3.0_
 
-_Jakarta™ Transaction Specification, Version 1.3_. Available at: _https://jakarta.ee/specifications/transactions/1.3_
+_Jakarta™ Transaction Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/transactions/2.0_
 
 _Java™ Transaction Service, Version 1.0_ (JTS specification)_. Available at: _https://www.oracle.com/technetwork/java/javaee/jts-spec095-1508547.pdf_
 
-_Jakarta™ Mail Specification, Version 1.6_. Available at: _https://jakarta.ee/specifications/mail/1.6_
+_Jakarta™ Mail Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/mail/2.0_
 
 _JavaBeans™ Activation Framework Specification Version 1.2 (JAF specification)_. Available at: _https://www.oracle.com/technetwork/java/javase/tech/index-jsp-138795.html_
 
-_Jakarta™ Connectors Specification, Version 1.7_. Available at: _https://jakarta.ee/specifications/connectors/1.7_
+_Jakarta™ Connectors Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/connectors/2.0_
 
-_Jakarta™ XML Web Services Specification, Version 2.3_. Available soon from: _https://jakarta.ee/specifications/xml-web-services/2.3_
+_Jakarta™ XML Web Services Specification, Version 3.0_. Available at: _https://jakarta.ee/specifications/xml-web-services/3.0_
 
 _Jakarta™ XML RPC Specification, Version 1.1_. Available at: _https://jakarta.ee/specifications/xml-rpc/1.1_
```

Comment:
At first, I was thinking that we should remove these entries that are removed from Jakarta EE 9.  But, since we do talk about them being removed, maybe it's still good to have pointers to the old specifications.  Just an observation...  Things like jax-rpc, jax-r, mgmt, deployment, distributed interop, corba, etc.

Reply:
Yes, I had the same thought